python/GafferSceneUI/EditScopeUI.py

```diff
@@ -101,3 +108,58 @@ def __pruningKeyPress( viewer, event ) :
 			GafferScene.EditScopeAlgo.setPruned( editScope, selection, True )
 
 	return True
+
+def __visibilityKeyPress( viewer, event ) :
+
+	if not ( event.key == "H" and event.Modifiers.Control ) :
+		return False
+
+	if not isinstance( viewer.view(), GafferSceneUI.SceneView ) :
+		return False
+
+	editScope = viewer.view().editScope()
+	if editScope is None or Gaffer.MetadataAlgo.readOnly( editScope ) :
+		return False
```

Comment:
Should be `return True` I think. The `False` returns above are good, because they give other handlers a chance in the case where we have no intention of ever doing anything. But in this case, we'd like to do something but can't - I think it would be confusing to the user if we let other handlers do something different in this case.

Reply:
Got it, that makes sense. Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5381/commits/ed1b6a582be422aa6c6a45587fda3bd794a284ba